shared/fs/browser/rows/rows.tsx

```diff
@@ -45,9 +45,11 @@ class Rows extends React.PureComponent<Props> {
           </WrapRow>
         )
       case RowTypes.RowType.Tlf:
+        console.log({songgao: 'rows', ...item})
```

Comment:
🔥 lets find a way to not let these slip through

Reply:
🤦🏻‍♂️sorry about that. Maybe I should put in a commit hook

Reply:
i have a different idea, pr soon